suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_cephadm.yaml

```diff
@@ -0,0 +1,656 @@
+#===============================================================================================
```

Comment:
Please match the `tier level to the one in Polarion... avoid matching to the stage in the QE pipeline.

Reply:
Done. Created two polarion test cases and added them to the last test in the suite as discussed.